index.html

```diff
@@ -41,6 +41,111 @@ <h3 class="connect"><a href="#">LET'S CONNECT</a></h3>
 				</ul>
 			</div>
 		</section>
+		<section class="project-works">
+			<ul class="work">
+				<li>
+					<img src="Assets/images/Portfolio2.png" alt="tonic project">
+					<h2>Tonic</h2>
+					<ul>
+						<li>CANOPY</li>
+						<li class="skill">Back-end dev</li>
+						<li class="year">2015</li>
+					</ul>
+					<p>
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programnming-languages">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<h2><a href="#">See Project</a></h2>
+				</li>
+			</ul>
```

Comment:
- [ ] You have misused the `ul` and `li` tags in the entire `project-works` section. `ul` is a wrapper tag for more than one `li` items. All the `work` and `big-project` class elements should have `li` tag. In this section, avoid the `ul` and `li` tags except for the one for the 'programming-languages` class element.  Kindly apply the same change for similar sections/elements. 
- [ ] Some typography and color are different from the original design in this section. Please stick to the original design. 

![Screenshot (4)](https://user-images.githubusercontent.com/64894927/120876014-22e43400-c5b7-11eb-8dda-1b18665110ac.png)  ![Screenshot (5)](https://user-images.githubusercontent.com/64894927/120876019-27105180-c5b7-11eb-87b7-ab00cadbcbac.png)

- [ ] There is also alignment problems in the projects section. Kindly fix this. 
![Screenshot (6)](https://user-images.githubusercontent.com/64894927/120876115-9ab25e80-c5b7-11eb-8180-03c9663412ff.png)